packages/components/Link/package.json

```diff
@@ -28,15 +28,19 @@
   "dependencies": {
     "@uifabricshared/foundation-compose": "^1.12.27",
     "@fluentui-react-native/adapters": ">=0.10.0 <1.0.0",
+    "@fluentui-react-native/framework": ">=0.8.26 <1.0.0",
     "@fluentui-react-native/interactive-hooks": ">=0.22.1 <1.0.0",
     "@fluentui-react-native/text": ">=0.19.0 <1.0.0",
     "@fluentui-react-native/tokens": ">=0.20.0 <1.0.0",
+    "@fluentui-react-native/use-styling": "^0.9.1",
     "@uifabricshared/foundation-composable": ">=0.11.1 <1.0.0",
-    "@uifabricshared/foundation-settings": ">=0.12.1 <1.0.0"
+    "@uifabricshared/foundation-settings": ">=0.12.1 <1.0.0",
+    "tslib": "^2.3.1"
```

Comment:
We need this directly?

Reply:
Not sure what it's there for, I kept it since it was in the experimental package.

Reply:
I would move it to devDependencies if possible.

Reply:
I dug, it's for: https://github.com/microsoft/fluentui-react-native/pull/1085/files